test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -60,7 +61,8 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		if ( dismissPageTemplateSelector ) {
 			await this.dismissPageTemplateSelector();
 		}
-		await this.dismissEditorWelcomeModal();
+		const editorWelcomeModal = new GuideComponent( this.driver, 4000 );
+		await editorWelcomeModal.dismissModal( '.components-guide' );
```

Comment:
This selector is not very specific. Is that because that is the only CSS hook available?

Reply:
To be honest, I haven't searched for the specific selector needed here. In fact I don't think it is necessary as the previous implementation was using the same exact selector. 
https://github.com/Automattic/wp-calypso/pull/50195/files/0ead51d42847d026454e5362cca24740faa9fb16#diff-b7f8735b885b065dc18c161fbf8de3c5bfecafd5fd04319842a8eea5f5c32180L682-L684